be/src/vec/exprs/table_function/vexplode_json_array.cpp

```diff
@@ -59,10 +61,12 @@ void VExplodeJsonArrayTableFunction<DataImpl>::process_row(size_t row_idx) {
 
     StringRef text = _text_column->get_data_at(row_idx);
     if (text.data != nullptr) {
-        rapidjson::Document document;
```

Comment:
will result compability issue？ eg. upgrade fe but be not upgrate be

Reply:
I see, we should keep string can also be kept